app/components/MemberForm/CreateMemberButton/CreateMemberButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { PlusIcon } from '@camp/icons';
+import { messages } from '@camp/messages';
+import { createTestAttr } from '@camp/test';
+import { Button } from '@mantine/core';
+
+import { createMemberButtonId as ids } from './CreateMemberButton.ids';
+
+interface Props {
+  variant?: 'filled' | 'outline';
+  onAddNewMember?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
It's better to be onClick, since this component only has design concerns.
This callback is not used for adding a new member it's just a normal click handler.

Reply:
U mean changing the name?

Reply:
Yes